sympy/physics/wigner.py

```diff
@@ -194,12 +194,34 @@ def wigner_3j(j_1, j_2, j_3, m_1, m_2, m_3):
 
     - Jens Rasch (2009-03-24): initial version
     """
-    if int(j_1 * 2) != j_1 * 2 or int(j_2 * 2) != j_2 * 2 or \
-            int(j_3 * 2) != j_3 * 2:
+
+    def convert_float_to_rational_if_half_integer(value):
+        if isinstance(value, float):
+            fraction_value = Fraction(value).limit_denominator()
+            if fraction_value.denominator == 2:
+                return fraction_value
+        return value
+
+    # Example list of values
```

Comment:
why `Example`? These are the inputs of the method and not an example :-)

Reply:
Yes I left the comment while I was testing the code for myself using example values , my bad sorry